www/includes/easyparliament/templates/html/index.php

```diff
@@ -0,0 +1,162 @@
+    <div class="hero">
+        <div class="row">
+        <div class="hero__mp-search">
+            <div class="hero__mp-search__wrap">
+                <?php if (count($mp_data)) { ?>
+                <h1>Does <?= $mp_data['name'] ?> represent you?</h1>
```

Comment:
I note that the "former" was lost in 4eb267b5fdbf1dba3faa719dab40502fe91c03ca and so this question doesn't make much sense during an electoral period; something needs to be done when former is present.